Vagrantfile

```diff
@@ -3,19 +3,20 @@
 
 require 'yaml'
 
-ansible_vars = YAML.load_file("provisioning/vars.yml")
+common_ansible_vars = YAML.load_file("provisioning/vars.yml")
+vagrant_ansible_vars = YAML.load_file("provisioning/vagrant-vars.yml")
 
-app_name = ansible_vars["nodejs_app_name"]
+app_name = common_ansible_vars["nodejs_app_name"]
 
-app_directory = ansible_vars["nodejs_app_install_dir"]
+app_directory = vagrant_ansible_vars["nodejs_app_install_dir"]
```

Comment:
I'm fine with this - I renamed the variables in the Vagrantfile, reran `Vagrant up` and ran all the tests again to make sure I didn't miss a crucial rename. All good.